clai/README.md

```diff
@@ -51,10 +51,73 @@ Either way, running `clai` will start an interactive session where you can chat
 - `/multiline`: Toggle multiline input mode (use Ctrl+D to submit)
 - `/cp`: Copy the last response to clipboard
 
+## Web Chat UI
+
+Launch a web-based chat interface for your agent:
+
+```bash
+clai web --agent module:agent_variable
+```
+
+For example, if you have an agent defined in `my_agent.py`:
+
+```python
+from pydantic_ai import Agent
+
+my_agent = Agent('openai:gpt-5', system_prompt='You are a helpful assistant.')
+```
+
+Launch the web UI with:
+
+```bash
+clai web --agent my_agent:my_agent
+```
+
+This will start a web server (default: http://127.0.0.1:7932) with a chat interface for your agent.
+
+### Web Command Options
+
+- `--agent`, `-a`: Agent to serve in `module:variable` format
+- `--models`, `-m`: Comma-separated models to make available (e.g., `gpt-5,sonnet-4-5`)
+- `--tools`, `-t`: Comma-separated builtin tool IDs to enable (e.g., `web_search,code_execution`)
```

Comment:
Link to builtin tools docs please. We may also need to list all the IDs as I don't think they're documented anywhere

Reply:
Is "to enable" correct? Are they all enabled by default or just available as options?

Reply:
most correctest would probably be `offer` or `put at the disposal` I guess, but enable seems good enough `enable for the user`